checks.d/etcd.py

```diff
@@ -68,6 +68,17 @@ def check(self, instance):
         # Load values from the instance config
         url = instance['url']
         instance_tags = instance.get('tags', [])
+
+        ssl_params = {
+            'ssl': instance.get('ssl', False),
+            'ssl_keyfile': instance.get('ssl_keyfile', None),
+            'ssl_certfile': instance.get('ssl_certfile', None),
+            'ssl_cert_reqs':  instance.get('ssl_cert_reqs', True)
```

Comment:
We probably cannot use `_is_affirmative` for the `'ssl_cert_reqs'` as it can be used to pass in the location of the certificate authority certificate.  Unless you think we should just change it over to be explicitly for the CA file.